diff -Naru podman-2.1.0.orig/libpod/image/docker_registry_options.go podman-2.1.0/libpod/image/docker_registry_options.go
--- podman-2.1.0.orig/libpod/image/docker_registry_options.go	2020-09-30 20:37:48.608273546 +0200
+++ podman-2.1.0/libpod/image/docker_registry_options.go	2020-09-30 20:38:53.112273538 +0200
@@ -1,12 +1,12 @@
 package image
 
 import (
-	"fmt"
+	//"fmt"
 
 	"github.com/containers/buildah/pkg/parse"
 	"github.com/containers/image/v5/docker/reference"
 	"github.com/containers/image/v5/types"
-	podmanVersion "github.com/containers/podman/v2/version"
+	//podmanVersion "github.com/containers/podman/v2/version"
 )
 
 // DockerRegistryOptions encapsulates settings that affect how we connect or
@@ -67,7 +67,7 @@
 	}
 	sc.AuthFilePath = authFilePath
 	sc.DirForceCompress = forceCompress
-	sc.DockerRegistryUserAgent = fmt.Sprintf("libpod/%s", podmanVersion.Version)
+	//sc.DockerRegistryUserAgent = fmt.Sprintf("libpod/%s", podmanVersion.Version)
 
 	return sc
 }
